views/install.sh.erb

```diff
@@ -311,7 +313,7 @@ do_wget() {
 # do_curl URL FILENAME
 do_curl() {
   echo "trying curl..."
-  curl -sL -D $tmp_dir/stderr "$1" > "$2"
+  curl -sL --connect-timeout $download_timeout -D $tmp_dir/stderr "$1" > "$2"
```

Comment:
@lamont-granquist From my bit of testing on curl 7.19.7 the `--connect-timeout` includes DNS lookups; if the DNS server doesn't respond before the given timeout curl will fail.  It would be good to test this on other versions of curl though.